pynxtools/dataconverter/helpers.py

```diff
@@ -32,7 +33,7 @@
 from pynxtools.nexus import nexus
 from pynxtools.nexus.nexus import NxdlAttributeError
 
-logger = logging.getLogger(__name__)
+pynx_logger = logging.getLogger(__name__)
```

Comment:
This is inconsistent. In other places you use the central logger and here you use a local logger.

Reply:
That's came from rebase. I will fixed
 it